providers/fab/src/airflow/providers/fab/www/extensions/init_wsgi_middlewares.py

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from airflow.configuration import conf
+from airflow.exceptions import AirflowConfigException
+from werkzeug.middleware.proxy_fix import ProxyFix
+
+if TYPE_CHECKING:
+    from flask import Flask
+
+
+def init_wsgi_middleware(flask_app: Flask) -> None:
+    """Handle X-Forwarded-* headers and base_url support."""
+
+    # Apply ProxyFix middleware
+    if conf.getboolean("webserver", "ENABLE_PROXY_FIX"):
```

Comment:
We removed this config because it was no longer used. Please update `airflow-core/src/airflow/cli/commands/config_command.py` as well to no longer mark this config as removed.

Reply:
I think it would be better to move this config to `fab` instead of keeping it in `webserver` but I think it is too late since we already cut 3.0.0. Unless we consider it a bug fix and put it in 3.0.X branch, then we could still rename this config? Maybe @potiuk knows better on that question

Reply:
Completly agree, I moved the configuration from webserver to fab and added the documentation in the provider docs.